physics/mass_energy_equivalence.py

```diff
@@ -0,0 +1,42 @@
+"""
+Mass energy Equivalene
+Reference: https://en.wikipedia.org/wiki/Mass%E2%80%93energy_equivalence
+"""
+import math as m
+
+""" Function to calculate energy(in joules) equivalent of a given
+mass(in kilograms) using Einstein's equation E=mc^2"""
+
+
+def calculate_energy(mass: float) -> float:
+    """
+    get energy equivalent of given mass in joules:
+    >>> calculate_energy(1.2)
+    1.0785062144841811e+17
+    >>> calculate_energy(3.0)
+    2.6962655362104528e+17
+    >>> calculate_energy(5)
+    4.493775893684088e+17
+    >>> calculate_energy(-5)
+    Traceback (most recent call last):
+      ...
+        raise TypeError("mass cannot negative, less than 0")
+    TypeError: mass cannot negative, less than 0
+    """
+    if mass < 0:
+        raise TypeError("mass cannot negative, less than 0")
+    # Speed of light in meters per second (approximately)
+    speed_of_light = 299792458
```

Comment:
1. Could you pull this out of the function as a constant?
2. 299,792,458 m/s is actually the _exact_ value of the speed of light, not an approximation

Reply:
I have defined the constant speed of light outside the function.